routes/web/assisted.php

```diff
@@ -9,6 +9,9 @@
 Route::group(['middleware' => ['permission:update-assisted']], function () {
     Route::get('/{assisted}/edit', 'AssistedEdit')->name('assisteds.edit');
     Route::put('/{assisted}', 'AssistedUpdate')->name('assisteds.update');
+
+    Route::get('/{assisted}/familyIncome', 'AssistedFamilyIncomeEdit')->name('assistedsFamilyIncomes.edit');
+    Route::put('/{assisted}', 'AssistedFamilyIncomeUpdate')->name('assistedsFamilyIncomes.update');
```

Comment:
```suggestion
    Route::put('/{assisted}/familyIncome', 'AssistedFamilyIncomeUpdate')->name('assistedsFamilyIncomes.update');
```

Está ocorrendo um conflito de rotas, pois a rota é igual à rota `assisteds.update` a requisição estava caindo nessa primeira rota pois ela aparece antes no arquivo de rotas.